lib/view-service-stack.js

```diff
@@ -1,17 +1,361 @@
-const cdk = require('@aws-cdk/core');
-
-class ViewServiceStack extends cdk.Stack {
-  /**
-   *
-   * @param {cdk.Construct} scope
-   * @param {string} id
-   * @param {cdk.StackProps=} props
-   */
-  constructor(scope, id, props) {
-    super(scope, id, props);
-
-    // The code that defines your stack goes here
-  }
+const {Construct, Duration, RemovalPolicy, Stack, StackProps} = require("@aws-cdk/core");
```

Comment:
this is nitpicking, but it would be nice to use single-quote strings here to be consistent with rest of script

Reply:
Thanks, fixed!